device-types/Huawei/CE8861-4C-EI.yaml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Huawei
+model: CE8861-4C-EI
```

Comment:
Is this a parent device (i.e. other devices get put into this chassis)? If so, please add `subdevice_role: parent` since this appears to be a parent chassis device

Reply:
The device does not contain sub-devices, only modules. No device-bays so no requirement for `subdevice_role`